client/src/core/client/stream/shadow/EmotionShadowRoot.tsx

```diff
@@ -1,39 +0,0 @@
-import createCache from "@emotion/cache";
```

Comment:
Dang, didn't even realize this was a thing. Wild.

Reply:
same! wild indeed.